config.json

```diff
@@ -1457,57 +1617,99 @@
+        "version": "1.2.0",
+        "nugetVersion": "1.2.0",
+        "nugetId": "Xamarin.AndroidX.Wear.Compose.Material.Core",
+        "dependencyOnly": false
+      },
       {
         "groupId": "androidx.wear.compose",
         "artifactId": "compose-navigation",
-        "version": "1.1.2",
-        "nugetVersion": "1.1.2.3",
+        "version": "1.2.0",
+        "nugetVersion": "1.2.0",
         "nugetId": "Xamarin.AndroidX.Wear.Compose.Navigation",
         "dependencyOnly": false
       },
+      {
+        "groupId": "androidx.wear.protolayout",
+        "artifactId": "protolayout",
+        "version": "1.0.0",
+        "nugetVersion": "1.0.0",
+        "nugetId": "Xamarin.AndroidX.ProtoLayout",
+        "dependencyOnly": false
+      },
+      {
+        "groupId": "androidx.wear.protolayout",
+        "artifactId": "protolayout-expression",
+        "version": "1.0.0",
+        "nugetVersion": "1.0.0",
+        "nugetId": "Xamarin.AndroidX.ProtoLayout.Expression",
+        "excludedRuntimeDependencies": "org.jetbrains.kotlin.kotlin-stdlib",
+        "dependencyOnly": false
+      },
+      {
+        "groupId": "androidx.wear.protolayout",
+        "artifactId": "protolayout-expression-pipeline",
+        "version": "1.0.0",
+        "nugetVersion": "1.0.0",
+        "nugetId": "Xamarin.AndroidX.ProtoLayout.Expression.Pipeline",
+        "dependencyOnly": false
+      },
+      {
+        "groupId": "androidx.wear.protolayout",
+        "artifactId": "protolayout-proto",
+        "version": "1.0.0",
+        "nugetVersion": "1.0.0",
+        "nugetId": "Xamarin.AndroidX.ProtoLayout.Proto",
+        "dependencyOnly": false
+      },
       {
         "groupId": "androidx.wear.tiles",
         "artifactId": "tiles",
-        "version": "1.1.0",
-        "nugetVersion": "1.1.0.3",
+        "version": "1.2.0",
+        "nugetVersion": "1.2.0",
         "nugetId": "Xamarin.AndroidX.Wear.Tiles",
         "dependencyOnly": false
       },
       {
         "groupId": "androidx.wear.tiles",
         "artifactId": "tiles-material",
-        "version": "1.1.0",
-        "nugetVersion": "1.1.0.3",
+        "version": "1.2.0",
+        "nugetVersion": "1.2.0",
         "nugetId": "Xamarin.AndroidX.Wear.Tiles.Material",
         "dependencyOnly": false
       },
       {
         "groupId": "androidx.wear.tiles",
         "artifactId": "tiles-proto",
-        "version": "1.1.0",
-        "nugetVersion": "1.1.0.3",
+        "version": "1.2.0",
+        "nugetVersion": "1.2.0",
         "nugetId": "Xamarin.AndroidX.Wear.Tiles.Proto",
         "dependencyOnly": false
       },
       {
         "groupId": "androidx.wear.tiles",
         "artifactId": "tiles-renderer",
-        "version": "1.1.0",
-        "nugetVersion": "1.1.0.3",
+        "version": "1.2.0",
+        "nugetVersion": "1.2.0",
         "nugetId": "Xamarin.AndroidX.Wear.Tiles.Renderer",
+        "excludedRuntimeDependencies": "androidx.wear.protolayout.protolayout-renderer",
```

Comment:
I guess this is to avoid the Guava `31.1-jre` dependency, but without fulfilling this dependency the package won't work.  Another option would be to add `frozen: true` to the `1.1.0` versions and not update these packages until we have a way to deal with the dependency.